homeassistant/components/rachio/switch.py

```diff
@@ -58,11 +67,14 @@ def _create_entities(hass, config_entry):
     for controller in person.controllers:
         entities.append(RachioStandbySwitch(controller))
         zones = controller.list_zones()
+        schedules = controller.list_schedules()
         current_schedule = controller.current_schedule
-        _LOGGER.debug("Rachio setting up zones: %s", zones)
         for zone in zones:
             _LOGGER.debug("Rachio setting up zone: %s", zone)
             entities.append(RachioZone(person, controller, zone, current_schedule))
+        for sched in schedules:
+            _LOGGER.debug("Added schedule: %s", sched)
```

Comment:
If it's still enough info, it would save on the logging calls if we just logged the list with added entities once. The entity has a repr magic method and will print the name and current state.